core/integration/features/rewards/return_volatility_rewards.feature

```diff
@@ -0,0 +1,405 @@
+
+    Then the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference |
+      | party2 | ETH/DEC21 | sell | 10     | 1003  | 0                | TYPE_LIMIT | TIF_GTC | p2-sell   |
+      | aux2   | ETH/DEC21 | buy  | 10     | 1003  | 1                | TYPE_LIMIT | TIF_GTC |           |
+
+    Then the network moves ahead "1" epochs
+
+    Then the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference |
+      | party1 | ETH/DEC21 | sell | 2      | 1003  | 0                | TYPE_LIMIT | TIF_GTC | p1-buy1   |
+      | aux1   | ETH/DEC21 | buy  | 2      | 1003  | 1                | TYPE_LIMIT | TIF_GTC |           |
+
+    Then the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference |
+      | party2 | ETH/DEC21 | buy  | 3      | 1004  | 0                | TYPE_LIMIT | TIF_GTC | p2-sell   |
+      | aux2   | ETH/DEC21 | sell | 3      | 1004  | 1                | TYPE_LIMIT | TIF_GTC |           |
+
+    Then the network moves ahead "1" epochs
+
+    # looking at the returns of all parties for a window of 2:
+    # aux1 = [4,1] => variance = 4.5
+    # aux2 = [0,0] => N/A
+    # party1 = [2,1] => variance = 0.5 => however party1 has insufficient stake so isn't eligible
+    # party2 = [0] => N/A
+    And "a3c024b4e23230c89884a54a813b1ecb4cb0f827a38641c66eeca466da6b2ddf" should have general account balance of "990000" for asset "VEGA"
+    And "aux1" should have vesting account balance of "10000" for asset "VEGA"
+
+  Scenario: eligible party with average notional less than threshold doesn't get a reward (0056-REWA-077)
+    # setup recurring transfer to the reward account - this will start at the end of this epoch
+    Given the parties submit the following recurring transfers:
+      | id | from                                                             | from_account_type    | to                                                               | to_account_type                       | asset | amount | start_epoch | end_epoch | factor | metric                            | metric_asset | markets | lock_period | window_length | distribution_strategy | entity_scope | individual_scope | staking_requirement | notional_requirement |
+      | 1  | a3c024b4e23230c89884a54a813b1ecb4cb0f827a38641c66eeca466da6b2ddf | ACCOUNT_TYPE_GENERAL | 0000000000000000000000000000000000000000000000000000000000000000 | ACCOUNT_TYPE_REWARD_RETURN_VOLATILITY | VEGA  | 10000  | 1           |           | 1      | DISPATCH_METRIC_RETURN_VOLATILITY | ETH          |         | 2           | 2             | PRO_RATA              | INDIVIDUALS  | ALL              | 0                   | 10000                |
+
+    Then the network moves ahead "1" epochs
+
+    Given time is updated to "2023-09-23T00:00:30Z"
```

Comment:
Is there a reason why we're manually setting time to a specific value here? It's preferable to avoid this...

Reply:
yes there's a very good reason we do. we want position to be recorded at this time, which is with respect to the epoch start time the mid point of the epoch.

Reply:
makes sense, alrighty